pages/docs/community/styleguide/grammar.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Grammar
+description: This document provides grammar usage guidelines for clear and effective technical writing.
+---
+## Grammar
+
+This style guide defines standards for AsyncAPI documentation for grammar across topics such as abbreviations and acronyms, active voice, capitalization, spelling, and verb tense. If you are a part of the AsyncAPI Slack, use the #13_docs channel for any queries. Please make sure to go through the style guide before contributing to AsyncAPI documentation.
+
+## Abbreviations and acronyms
+
+When you use an abbreviation or acronym for the first time, spell it out and include the shortest form in parentheses. 
+
+Abbreviations are meant to be used to save the reader’s time. They make your documentation look clean. However, using abbreviations may come with some downsides. Abbreviations add a layer of abstraction to text, which requires the reader to spend some mental energy to expand. Thus, it is best to add abbreviations in case of words that have been repeatedly used throughout your document. Abbreviations or acronyms, once defined in a document, should be used consistently to ensure clarity and avoid confusion for readers.
```

Comment:
```suggestion
Abbreviations are meant to be used to save the reader’s time. They make your documentation look clean. However, using abbreviations may come with some downsides. Abbreviations add a layer of abstraction to text, which requires the reader to spend some mental energy to expand. Thus, it is best to add abbreviations in case of words that have been repeatedly used throughout your document. Once defined in a document, abbreviations or acronyms should be used consistently to ensure clarity and avoid confusion for readers
```